runtime/hsa-runtime/core/driver/kfd/amd_kfd_driver.cpp

```diff
@@ -58,6 +58,25 @@
 namespace rocr {
 namespace AMD {
 
+static_assert(
+    (sizeof(core::ShareableHandle::handle) >= sizeof(amdgpu_bo_handle)) &&
+        (alignof(core::ShareableHandle::handle) >= alignof(amdgpu_bo_handle)),
+    "ShareableHandle cannot store a amdgpu_bo_handle");
+
+static __forceinline uint64_t drm_perm(hsa_access_permission_t perm) {
```

Comment:
Would it be better to put this in an anonymous namespace?

Reply:
Sure, they're equivalent (static was un-deprecated for this use).